pom.xml

```diff
@@ -41,6 +50,27 @@
                 <configuration>
                     <source>1.8</source>
                     <target>1.8</target>
+                    <compilerArgument>-Xlint:unchecked</compilerArgument>
```

Comment:
Why is this necessary?

Reply:
It helps to know where unchecked assignments are happening in code as a matter of best practice code smell reduction.

Reply:
Ah, what IDE are you using (if at all)? IntelliJ can detect unchecked assignments; I believe it's one of the checks enabled by default. Either way, good to know. I had not seen the `Xlint` flag before.

Reply:
I use Eclipse, or just Sublime Text, depending on what I'm trying to accomplish.